Makefile

```diff
@@ -22,8 +22,12 @@ include $(BOLOS_SDK)/Makefile.defines
 
 APP_PATH = ""
 # All but bitcoin app use dependency onto the bitcoin app/lib
+ifeq ($(COIN),zencash)
```

Comment:
Please move this into zencash's own conditional block line 87

Reply:
Update: The requested change is now to remove this `ifdef` all together.

Reply:
This has been done by removing 9478e005ab1e34aa386700840facb0ef1821926f